sound/soc/intel/boards/sof_sdw_max98373.c

```diff
@@ -0,0 +1,73 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+// Copyright (c) 2020 Intel Corporation
+
+/*
+ *  sof_sdw_max98373 - Helpers to handle 2x MAX98373 codec devices
+ *  from generic machine driver
+ */
```

Comment:
Mark Brown wants us to use the // style for the .c files. We missed it in the past, let's follow his guidance for this one.

Reply:
Sure, lets do it..